### PR TITLE
if/else/for/do/while highlighting fixed

### PR DESCRIPTION
**Pull request by bshtv**
Screenshot at Jan 26 17-35-46

### CODE REVIEW

**Comment by asm0dey** [comment]
LGTM, please mention me when this is redy

File: ki-shell/pom.xml [hunk 1]
```diff
@@ -4,7 +4,7 @@
   <parent>
     <groupId>org.jetbrains.kotlinx</groupId>
     <artifactId>ki</artifactId>
-    <version>0.4.5-SNAPSHOT</version>
+    <version>0.4.5</version>
```
Comment (ligee) [inline ki-shell/pom.xml:7]: Please do not forget to increment version before the next release.

File: ki-shell/src/main/kotlin/org/jetbrains/kotlinx/ki/shell/parser/KotlinParserListenerForHighlighting.kt [hunk 2]
```diff
@@ -6,6 +6,11 @@ import org.antlr.v4.runtime.tree.ErrorNode
 import org.antlr.v4.runtime.tree.TerminalNode
 import org.jetbrains.kotlinx.ki.shell.parser.KotlinParser.*
 
+private const val IF_EXPR_OFFSET = 2
```
Comment (ligee) [inline ki-shell/src/main/kotlin/org/jetbrains/kotlinx/ki/shell/parser/KotlinParserListenerForHighlighting.kt:9]: Please add the source of the magic constants, i.e. "these offsets are taken from official grammar <url> for navigating to the sub-elements, not directly represented in it.

**Review by ligee — CHANGES_REQUESTED** [review]

File: ki-shell/pom.xml (hunk 1 above)
Comment (asm0dey) [reply ki-shell/pom.xml:7]: BTW, thanks, there is no need to change versions manually, version release plugin does it for us

File: ki-shell/src/main/kotlin/org/jetbrains/kotlinx/ki/shell/parser/KotlinParserListenerForHighlighting.kt (hunk 2 above)
Comment (bshtv) [reply ki-shell/src/main/kotlin/org/jetbrains/kotlinx/ki/shell/parser/KotlinParserListenerForHighlighting.kt:18]: Done

**Review by ligee — APPROVED** [review]

**Pull request merged**